src/masternode-budget.cpp

```diff
@@ -1264,7 +1273,7 @@ void CBudgetManager::Sync(CNode* pfrom, const uint256& nProp, bool fPartial)
 
 bool CBudgetManager::UpdateProposal(const CBudgetVote& vote, CNode* pfrom, std::string& strError)
 {
-    LOCK(cs);
+    LOCK(cs_proposals);
```

Comment:
What about `mapOrphanMasternodeBudgetVotes`?
In `UpdateProposal ` is guarded by `cs_proposals` and in `CheckOrphanVotes` is guarded by `cs`.

Same applies for `askedForSourceProposalOrBudget` in different methods.

Reply:
mapOrphan* maps are guarded by `cs_votes`/`cs_finalizedvotes` introduced in a subsuequent commit